src/containers/Options/rewards.tsx

```diff
@@ -108,7 +109,7 @@ export default function Rewards({ route, navigation }) {
                   <CyDView>
                     <CyDText className='text-[14px] font-bold text-n300'>
                       <LoaderWithText
-                        value={rewardPoints.spend.toString()}
+                        value={rewardPoints.spend.toLocaleString('en-US')}
```

Comment:
don't enforce to en-US use toLocaleString() this will convert to the users local format he is in